sklearn/preprocessing/data.py

```diff
@@ -441,9 +442,13 @@ class PolynomialFeatures(BaseEstimator, TransformerMixin):
 
     Attributes
     ----------
+    n_input_features_ : int
+        The total number of input features.
 
-    powers_ :
```

Comment:
I agree. I have code that uses this attribute; it's not a huge deal to continue storing it.